src/lib.rs

```diff
@@ -74,41 +86,73 @@ use errors::*;
 
 mod diff;
 
-/// Assertions for a specific command
+/// Assertions for a specific command.
 #[derive(Debug)]
 pub struct Assert {
     cmd: Vec<String>,
-    expect_success: bool,
+    expect_success: Option<bool>,
```

Comment:
`Option<bool>` is more expressive than `bool`. Because `None` is the I-don't-care-case (e.g. should print "foo", but may fail or succeed).

If you don't like this change I'd suggest to change the docs instead. The call `.succeeds()` is really all over the place and could then be omitted in almost all cases except the doc-comment for `.succeeds()` itself, where it should state that this is pure decoration (unless you receive a partial assertion builder from somewhere). :-)